client/my-sites/stats/activity-log-confirm-dialog/index.jsx

```diff
@@ -113,7 +113,7 @@ class ActivityLogConfirmDialog extends Component {
 								className="activity-log-confirm-dialog__more-info-link"
 								href="https://help.vaultpress.com/one-click-restore/"
 							>
-								<Gridicon icon="chat" />
+								<img src="/calypso/images/illustrations/AnyQuestions.png" />
```

Comment:
The orange pumpkins should be blue to follow our WordPress.com design handbook.

Reply:
Thanks @drw158! I wonder about that though because this is in the Jetpack section - should they not be Jetpack green?

Reply:
I'm not sure if this is going to scale well.